Simplenote/src/main/java/com/automattic/simplenote/utils/SearchTokenizer.java

```diff
@@ -10,6 +10,7 @@
     static public final char SINGLE_QUOTE = '\'';
     static public final char GLOB = '*';
     static public final char COLON = ':';
+    static public final char COMMA = ',';
```

Comment:
hm. some of these make sense. colon and comma seem a little silly IMO. did you find this strange when you found them?

Reply:
COLON is used in the `SearchTokenizer#parseQuery` which returns a string that is then used to build the query condition. These constants are also used outside the SearchTokenizer. That's why I thought to define the COMMA within that same class. However, since COMMA is not used within the SearchTokenizer, I could move the COMMA variable to `TagsMultiAutoCompleteTextView`. Let me know your suggestions.

Reply:
probably best not to change things. I wasn't so much questioning the use of constants so much as asking if these constants are pointless. kind of like `static public final int ONE = 1`

let's just leave it. I would be personally fine if we didn't create a constant for comma; it's not a name of something other than the thing itself, and any other value would break the semantic of the constant. so if you want to keep it as is that's fine but it'd also be fine if you inlined `','` instead of creating the constant.

Reply:
Maybe the intention was to facilitate refactoring by reuse. Anyway, I just inlined the character literals. as there is no obvious benefit from doing otherwise.